invenio_app_ils/ui/src/invenio_app_ils/common/api/eitems/eitem.js

```diff
@@ -46,10 +46,29 @@ const queryBuilder = () => {
   return new QueryBuilder();
 };
 
+const count = query => {
```

Comment:
This block of code gets repeated for every query builder and is identical

Reply:
As for get or similar methods, I think we will have to live with that, unless you have suggestions! :)